nextcord/enums.py

```diff
@@ -2037,6 +2039,32 @@ class ForumLayoutType(IntEnum):
     """Display posts as a collection of posts with images, this is more image focused."""
 
 
+class IntegrationType(IntEnum):
```

Comment:
Both class docstrings (and their parameters' docstrings) added in this file are missing periods at the ends.